var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -250,6 +250,6 @@ def write_rpath_specs(self):
             for line in lines:
                 out.write(line + '\n')
                 if line.startswith('*link:'):
-                    out.write(r'-rpath {0}:{1} \n'.format(
```

Comment:
You're welcome.  That shaves epsilon off of the balance (thanks for all the help from your side)....